cmd/validate_editorconfig.go

```
@@ -0,0 +1,194 @@
+)
+
+var editorConfigCmd *cobra.Command = &cobra.Command{
+	Use:   "editorconfig",
+	Short: "Validate all files against the EditorConfig",
+	Long:  "Validate all files against the project's EditorConfig rules",
+	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		initializeConfig(cmd)
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		if cliConfig.Help {
+			cmd.Help()
```

golangci-lint (1.62.2)

Error return value of `cmd.Help` is not checked

(errcheck)